__init__.py

```diff
@@ -214,6 +224,29 @@ def draw(self, context):
         addon_updater_ops.update_notice_box_ui(self, context)
 
 
+class POSE_PT_CobraTools(bpy.types.Panel):
+    """Creates a Panel in the scene context of the properties editor"""
+    bl_label = "Cobra Rig Tools"
+    bl_space_type = 'PROPERTIES'
+    bl_region_type = 'WINDOW'
+    bl_context = "data"
+
+    @classmethod
+    def poll(cls, context):
+        if not context.armature:
+            return False
+        return True
+
+    def draw(self, context):
+        addon_updater_ops.check_for_update_background()
+        layout = self.layout
+        icon = preview_collection["frontier.png"].icon_id
+        row = layout.row(align=True)
+        sub = row.row()
+        sub.operator("pose.generate_rig_edit", icon_value=icon)
+        addon_updater_ops.update_notice_box_ui(self, context)
```

Comment:
needed here?

Reply:
I don't know what this does but when I removed the row.row and layout.row lines the button disappeared

Reply:
The whole function is responsible for drawing the panel. I was referring to `addon_updater_ops.update_notice_box_ui(self, context)`, which takes care of running the addon updater. Only needs to be mentioned once per addon IIRC.

Reply:
removed unnecessary line